Zilon.Core/Zilon.Core.Benchmarks.Move/MoveBench.cs

```diff
@@ -5,23 +5,22 @@
 
 using Microsoft.Extensions.DependencyInjection;
 
-using Zilon.Core.Benchmark;
 using Zilon.Core.Client;
 using Zilon.Core.Commands;
-using Zilon.Core.Persons;
 using Zilon.Core.Players;
 using Zilon.Core.Schemes;
 using Zilon.Core.Tactics;
 using Zilon.Core.Tactics.Behaviour;
 using Zilon.Core.Tactics.Spatial;
 using Zilon.Core.Tests.Common;
-using Zilon.Core.Tests.Common.Schemes;
+using Zilon.Core.World;
 
 namespace Zilon.Core.Benchmarks.Move
 {
     public class MoveBench
     {
         private ServiceProvider _serviceProvider;
+        private IGlobe globe;
```

Comment:
Codacy found an issue: [Remove the field 'globe' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6388157)